questions/accessibility-aria.md

```diff
@@ -0,0 +1,36 @@
+### What is ARIA and when should you use it?
+
+#### Answer
+
+ARIA stands for "Accessible Rich Internet Applications". ARIA code is HTML. It provides additional attributes in the development of web applications to offer screen reader users more context about dynamic components, including the component's role, name, and state. It should only be used when an HTML element equivalent is not available or lacks full support. ARIA is semantic, and doesn't actually make your component dynamic.
```

Comment:
ARIA code is HTML and JavaScript isn't it? There's many things you can do with ARIA you can't do without JavaScript

Reply:
You're correct that ARIA is mostly useless without JavaScript, but ARIA itself is HTML attributes.

Reply:
I was torn about "WAI-ARIA" too. I think you're right that WAI-ARIA is the most correct way to teach people about it. I was thinking from the point of view of devs who hear it called ARIA and could feel more familiar with the question.

Reply:
I should probably think of it as a standard, too. The attributes are a way to create accessible web apps. The spec specifically states:
> WAI-ARIA is a technical specification that provides a framework to improve the accessibility and interoperability of web content and applications.

Reply:
This is why I think of it as code:
> The incorporation of WAI-ARIA is a way for an author to provide proper semantics for custom widgets to make these widgets accessible, usable, and interoperable with assistive technologies.